### DISCUSSION TITLE
Is it possible to tell Nefarious to prefer HDR downloads?

### DISCUSSION THREAD

**Discussion by jordyvandomselaar** [lardbit/nefarious#96] [Q&A]
Right now I have to use the manual search every time I'd like to use HDR, is there a way to automate that?

**Comment by lardbit** [comment]
Hey @jordyvandomselaar,

Sorry, I apparently don't get notified on new discussions for some reason. Yes, you can tell nefarious to download "ultra-hd" by selecting that quality profile either in the global settings or for a specific movie.  

Do you not see that option?

Global Settings:
![image](https://user-images.githubusercontent.com/45122868/111213032-58701700-859e-11eb-974e-6f47aaa19bd8.png)


Movie:
![image](https://user-images.githubusercontent.com/45122868/111212984-4a21fb00-859e-11eb-8faa-fc22dac206b7.png)

**Reply by jordyvandomselaar** [reply]
It's alright =)

I do have the ultra HD setting and it does download 2160P movies but not the HDR ones, I have to manually search for the HDR versions

**Reply by lardbit** [reply]
Oh, I guess I didn't know HDR was different. I can add that as a quality option. Do you know all the tags that usually identifies it?

**Reply by lardbit** [reply]
I based my initial options from sonarr and they didn't have it, but maybe they do now. I'll check

**Reply by jordyvandomselaar** [reply]
I'm not sure, right now I manually check if the name contains HDR. Might be an issue on Sonarr's end!

**Reply by lardbit** [reply]
This is what radarr is doing (the movie clone of sonarr) so I can just mimic it's regex.

https://github.com/Radarr/Radarr/blob/master/src/NzbDrone.Core/Parser/QualityParser.cs#L39-L40

It's basically only looking for "rawhd" or "raw*hd".  Simple enough.  If you can create a github issue for it I'll be able to add it to the list of things to implement.
